src/config/env.ts

```diff
@@ -47,7 +47,7 @@ envCheck(
 	"APP_ID",
 	"APP_URL",
 	"APP_KEY",
-	"WEBHOOK_SECRET",
+	"WEBHOOK_SECRETS",
```

Comment:
**IMPORTANT**
One thing, since you change the env name, you have to put the current value of the `WEBHOOK_SECRET` into `WEBHOOK_SECRETS` of the vault (or stash).

But the thing is, how do you 100% SURE that the value in lastpass is the value that being used in prod? I can't think of a way to verify it. And if that value in lastpass is wrong, we may lost some webhooks, and need to rush to generate a new secrete and put it in vault, very rushy.

Reply:
Alternative, you can keep the `WEBHOOK_SECRET`, but have a new env like `WEBHOOK_SECRET_VAULT`, and in the code try both.

Another benefits is you don't have to do `JSON.parse`.

Reply:
Yep, `WEBHOOK_SECRETS` is a string which has both the old and new webhook secrets.
I agree, there is no direct way to check the current webhook secret. However, I checked the history in Lastpass and it has not been changed since `2022-06-16 01:12:35`, so pretty sure its still the same. And I don't think we have any lost webhooks yet, don't have any complaints yet.
And as for the alternative, its more scalable with one variable which holds multiple secrets instead of having 2 separate variables. I have discussed with @mboudreau  and agreed not to use two variables.